.github/workflows/create-plugin-release.yml

```diff
@@ -3,9 +3,6 @@ name: Build and create pre-release for the EKS cluster plugin
 on:
```

Comment:
Check if linter can be run before build (linked to PR https://github.com/dataiku/dss-plugin-eks-clusters/pull/72)